packages/layout/src/components/cards/common/views/address/addressPage.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useState } from 'react';
+import { Content } from '../../../components/content';
+import { Footer } from '../../../components/card';
+import { ArrowButton } from '../../../../buttons/buttons';
+import { cardType } from '../../../common/interfaces';
```

Comment:
we can avoid the last 2 imports adding it to this one.
`import { cardType, AddressAPIType, cardTypeName } from '../../../common/interfaces';`